test/controllers/dashboard_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class DashboardControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
```

Comment:
I would recommend using single quotes when you don't have string interpolation
`'should get index'`

Reply:
These tests were automatically generated, I will modify them though, I remember this from the second assignment.